instrumentation/log4j/log4j-1.2/log4j-1.2.gradle

```diff
@@ -0,0 +1,24 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "log4j"
+    module = "log4j"
+    versions = "[1.2,)"
```

Comment:
Should we cap that on 2.0? Or it has different module?

Reply:
ya it's a different module, we could cap, but nice to leave unbounded where possible